dbfit-java/build.gradle

```diff
@@ -83,7 +83,70 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task deploymentProperties {
+        doLast {
+            ext {
+                delimiterType = 'normal'
+                delimiter = ';'
+                setUpTestScriptPath = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/"
+                setUpTestDbScript = setUpTestScriptPath + "create-databases-${project.name}.sql"
+                tearDownTestDbScript = setUpTestScriptPath + "drop-databases-${project.name}.sql"
+            }
+        }
+    }
+
+    // to be overriden when needed
+    ext.connectionPropertiesFiles = []
+
+    task connectionFileProperties {
+        doLast {
+            copyConnectionProperties '.'
+            def props = new Properties()
+            file(connectionPropertiesFiles[0]).withInputStream { props.load(it) }
+            ext {
+                userName = props.username
+                passWord = props.password
+                service = props.service
+                database = props.database
+            }
+        }
+    }
+
+    task dbConnectionProperties(dependsOn: [deploymentProperties]) {
+    }
+
+    task adminConnectionFileProperties {
+        doLast {
+            def adminConnPropsFile
+            if (project.hasProperty('adminConnectionPropertiesFile')) {
+                adminConnPropsFile = adminConnectionPropertiesFile
+            } else {
+                adminConnPropsFile = connectionPropertiesFiles[0]
```

Comment:
Not at the moment. If you think we should remove this condition then I'm fine with that.